js-plot/README.md

```diff
@@ -2,22 +2,214 @@
 
 Running C++ code in a web browser is all nice, but we really want to grab someones attention by visualizing something. In this blog we are going to make a plot from the results coming from our C++ code.
 
-To make a plot we need some data. In a [previous](../js-webapp) post we found the root of an equation using the Newton-Raphson algorithm implemented in C++ and compiled to a WebAsssembly module.
+To make a plot we need some data. In a [previous](TODO fix url ../js-webapp) post we found the root of an equation using the Newton-Raphson algorithm implemented in C++ and compiled to a WebAsssembly module.
 A single root value makes for a depressing plot. The Newton-Raphson algorithm uses iterations to find the root so we will capture the data of each iteration and plot those.
 
 Let's make changes to the C++ code to retrieve the data from the iterations.
 
-TODO add code snippets
+## Iterations
 
-TODO explain how wasm vector is converted to array
+We will add `iterations` public property to the NewtonRaphson which is a vector of iteration structs. So the `newtonraphson.hpp` becomes
```

Comment:
Please first mention the struct then the property. Can you also explain what we have in the struct?

Reply:
done